src/Model/Actions.re

```diff
@@ -183,7 +183,7 @@ type t =
   | Pane(Feature_Pane.msg)
   | PaneTabClicked(Feature_Pane.pane)
   | PaneCloseButtonClicked
-  | VimDirectoryChanged(string)
+  | DirectoryChanged(string)
```

Comment:
I debated whether or not I should add a duplicate variant to distinguish the vim  `:cd` from other possible directory changes, but looking where this is used, it doesn't seem super specific to vim. Let me know if you think I should change it.

Reply:
Seems reasonable - it is more general than Vim!